sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -754,4 +754,29 @@ class PlanParserSuite extends AnalysisTest {
       assertEqual(query2, Distinct(a.union(b)).except(c.intersect(d, isAll = true), isAll = true))
     }
   }
+
+  test("create/alter view as insert into table") {
+    intercept[ParseException](parsePlan("CREATE VIEW testView AS INSERT INTO jt VALUES(1, 1)"))
```

Comment:
Ur, is this a behavior change? Previously, this seems to be `AnalysisException` with `Operation not allowed: CREATE VIEW ... AS INSERT INTO`.

Also, please check the exception message always.

Reply:
@dongjoon-hyun For parser exceptions, we would have list of expected tokens which may change to make this a little fragile ? Thats why i didn't check the exact message text here.
if you want it checked, would it be better to put it in SQlQueryTestSuite ? Let me know what you think.

Reply:
Oh, that means a subtle behavior change. There is no more kind message like `Operation not allowed: `?

Reply:
@dongjoon-hyun Let me paste a sample error from the describe query test.
```
DESCRIBE
   FROM desc_temp1 a
     insert into desc_temp1 select *
     insert into desc_temp2 select *
-- !query 13 schema
struct<>
-- !query 13 output
org.apache.spark.sql.catalyst.parser.ParseException

mismatched input 'insert' expecting {<EOF>, '(', ',', 'ANTI', 'CLUSTER', 'CROSS', 'DISTRIBUTE', 'EXCEPT', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LATERAL', 'LEFT', 'LIMIT', 'NATURAL', 'ORDER', 'PIVOT', 'RIGHT', 'SELECT', 'SEMI', 'MINUS', 'SORT', 'UNION', 'WHERE', 'WINDOW'}(line 3, pos 5)

````
I am thinking, this is coming from antlr itself as it knows the expected grammar ?

Reply:
@dongjoon-hyun I will do a contains and look for "mismatched input" like we do in test "select hint syntax". hopefully its ok.